database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -364,10 +441,14 @@ protected String getEscapedTableName() {
       return tableName;
     }
 
+    protected String getEscapedSchemaName() {
+      return dbSchemaName;
```

Comment:
This is not really escaped schema name right? but we are returning as it is. I see the problem exists for the tableName too. We should just call this method as getSchemaName and add javadoc saying subclasses can override method to perform any escaping required.